install.sh

```diff
@@ -315,13 +300,49 @@ unzip() {
     local _file="$1"
     local _dir="$2"
 
-    tar -xzvf "$_file" -C "$_dir"
+    tar -xzf "$_file" -C "$_dir"
 }
 
 get_default_install_dir() {
     printf %s "${HOME}/.temporalio"
 }
 
+get_install_dir() {
+    local _dir
+    _dir="$(get_default_install_dir)"
+
+    while [ $# -gt 0 ]; do
+        case "$1" in
+        --dir)
+            _dir="$2"
+            shift
+            ;;
+        *) ;;
+
+        esac
+        shift
+    done
+
+    printf %s "$_dir"
+}
+
+prompt_for_path() {
+    local _dirbin="$1"
+
+    local _source="export PATH=\"\\\$PATH:$_dirbin\" >> ~/.bashrc"
```

Comment:
Should this be `.bash_profile`?

Reply:
with .bash_profile users will have to re-login for the command to take effect. With .bashrc they can just open a new terminal and have `temporal` working

Reply:
Yeah, I brushed up on this and bashrc is better here.